### PR TITLE
Update raspberry-pi.md

### PR DESCRIPTION
**Pull request by unfriend**
added linuxserver/mariadb to the instructions since the official mariadb image does not work with the raspberry 4 arm64v7

### CODE REVIEW

File: docs/getting-started/raspberry-pi.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ Big thank you to [Guy Sheffer](https://github.com/guysoft) for
 
 Download our [docker-compose.yml](https://dl.photoprism.org/docker/arm64/docker-compose.yml) file
 (right click and *Save Link As...* or use `wget`) to a folder of your choice, 
-change the [configuration](config-options.md) as needed, and run `sudo docker-compose up` to start PhotoPrism:
+change the [configuration](config-options.md) as needed, replace the image under mariadb: with linuxserver/mariadb and run `sudo docker-compose up` to start PhotoPrism:
 
 ```
 wget https://dl.photoprism.org/docker/arm64/docker-compose.yml
-sudo docker-compose up
+sudo docker-compose
```
Comment (alexislefebvre) [inline docs/getting-started/raspberry-pi.md:12]: Why this change?

Comment (unfriend) [reply docs/getting-started/raspberry-pi.md:12]: the official mariadb image seems to only build for arm64v8 and since the guide is for raspberry this fixes it, do you think it would be better do check with mariadb upstream? I just installed it this way this afternoon and it would not work with the version in the .yml file linked.

Comment (unfriend) [reply docs/getting-started/raspberry-pi.md:12]: I just realised my change was super stupid, must have been a typo on my side, sorry for wasting your time, will create a more meaningful pull request tomorrow, what i was meant to do was changing another line, might even create a new compose file.

Comment (unfriend) [reply docs/getting-started/raspberry-pi.md:12]: actually the cahnge makes sense, just removing the up was a typo.

Comment (alexislefebvre) [reply docs/getting-started/raspberry-pi.md:12]: `docker-compose` without argument only show the help on my computer.

Comment (unfriend) [reply docs/getting-started/raspberry-pi.md:12]: as i said, the first change in line 8 makes sense, the second one in line 12 where i remove the up was a typo, sorry

Comment (alexislefebvre) [reply docs/getting-started/raspberry-pi.md:12]: No problem, feel free to restore it. In a second commit, or by [amending the commit](https://docs.github.com/en/free-pro-team@latest/github/committing-changes-to-your-project/changing-a-commit-message#amending-older-or-multiple-commit-messages).

**Comment by lastzero** [comment]
Totally lost your PR out of sight... May https://github.com/photoprism/photoprism/issues/748 have been caused by this?

We're now using the alternative mariadb image in our example:
https://dl.photoprism.org/docker/arm64/docker-compose.yml
